src/game/default/g_client.c

```diff
@@ -1677,7 +1683,7 @@ void G_ClientBeginFrame(g_entity_t *ent) {
 	}
 
 	if (cl->locals.hook_think_time < g_level.time) {
-		G_ClientHookThink(ent);
+		G_ClientHookThink(ent, false);
```

Comment:
`Think` functions do not follow this prototype (the extra _Bool parameter), so we should rename this wrapper function, and keep the core `Think` function as-is.

Reply:
Ah, didn't know that.

Reply:
Still relevant.